scripts/agent-dev

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```

Comment:
nit: Since we're removing all scripts, I think we could also remove [this line](https://github.com/rancher/fleet/blob/master/.github/workflows/rancher-integration.yml#L21).